projects/go-lib/src/lib/components/go-off-canvas/go-off-canvas.component.html

```diff
@@ -6,17 +6,19 @@
 >
 </div>
 <div
-  [@offCanvas]="opened ? 'slideIn' : 'slideOut'"
+  [@offCanvas]="{value: opened ? 'slideIn' : 'slideOut', params: {width: size === 'large' ? '75vw' : '300px'}}"
   class="go-off-canvas"
+  [ngClass]="{large: size === 'large'}"
   role="dialog"
 >
   <div class="go-off-canvas__header">
     <h3 *ngIf="header" class="go-heading-3 go-heading--no-margin go-off-canvas__title">
       {{ header }}
     </h3>
-    <go-icon-button buttonSize="medium" buttonIcon="close"
+    <go-icon-button *ngIf="size === 'small'" buttonSize="medium" buttonIcon="close"
                     (handleClick)="closeOffCanvas()">
     </go-icon-button>
+    <ng-container *ngIf="size === 'large'" goPortalTarget="offCanvasHeader"></ng-container>
```

Comment:
Instead of performing two separate conditionals, should we switch this to use Angular's `ngIf/else` syntax?

Reply:
What if we end up with a "medium" size?

Reply:
I thought about that, if you think that is something that could eventually happen then maybe we should go with the `ngSwitchCase` approach, https://angular.io/api/common/NgSwitchCase